src/utils.js

```diff
@@ -68,9 +68,13 @@ function createElementNS( name ) {
 
 }
 
+function createElement( name ){
+	return document.createElement( name );
+}
+
 function createCanvasElement() {
 
-	const canvas = createElementNS( 'canvas' );
```

Comment:
> but an error was put into dev branch element creation.

Why is using `createElementNS()` here an error. It was implemented like this in purpose, see #9041.

Reply:
I understand the confusion, but I thought it only used in svg rendering. I saw that it was a relatively new commit, I thought it probably a typo with autocomplete. [Document: createElement() method - Web APIs | MDN (mozilla.org)](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) Incidentally, I am not sure the other pull request was correct. If I am in error, I apologize; trying to help.

Reply:
I don't think it's a typo...